.github/actions/run-airbyte-ci/action.yml

```diff
@@ -83,6 +83,10 @@ inputs:
     description: "Python registry URL to publish python package"
     default: "https://upload.pypi.org/legacy/"
     required: false
+  python_registry_check_url:
+    description: "Python registry URL to check whether a python package is published already"
+    default: "https://pypi.org/pypi"
+    required: false
```

Comment:
I'm not sure this input is required as in CI use cases there's no reason to change this URL at the moment. 
Same for `PYTHON_REGISTRY_URL`. The defaults are set at the airbyte-ci CLI level and we have to no reason to change these in the CI ATM. Am I correct?

Reply:
that's right, we can also remove them from here.